how_to_guides/set_up_status_cake.md

```diff
@@ -0,0 +1,79 @@
+# How to set up status cake with your service
+
+This guide outlines the basic requirements to set up status cake to monitor your website up time. Status cake works by making requests to your endpoint at given intervals and then sending a notification if it fails to receive a success HTTP status code. It can send notifications to email addresses and slack channels etc.
```

Comment:
Rather than 'it can' be should probably be more prescriptive and say 'This guide will include setting up notifications to email addresses and Slack channels, and the team may opt to include mobile phone alerts as well.'